src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -376,10 +376,10 @@ MXNET_OPERATOR_REGISTER_BINARY(_backward_tan)
 
 // arcsin
 MXNET_OPERATOR_REGISTER_UNARY(arcsin)
-.describe(R"code(Inverse sine, element-wise.
+.describe(R"code(Returns element-wise Inverse trigonometric sine value of the input.
```

Comment:
inverse should not be capitalized. @madjam jinx...

Same issue as above. "Trigonometric sine" is redundant". Also "value" is unnecessary. The context makes clear that it is a function and "Returns" makes clear that it returns a value.

Just say:

"Returns the inverse sine of the input array."